pkg/scaffold/resource/resource_test.go

```diff
@@ -94,38 +94,49 @@ var _ = Describe("Resource", func() {
 		It("should default the Resource by pluralizing the Kind", func() {
 			instance := &Resource{Group: "crew", Kind: "FirstMate", Version: "v1"}
 			Expect(instance.Validate()).To(Succeed())
+			Expect(instance.Init()).To(Succeed())
+			Expect(instance.Plural).To(Equal("firstmates"))
 			Expect(instance.Resource).To(Equal("firstmates"))
 
 			instance = &Resource{Group: "crew", Kind: "Fish", Version: "v1"}
 			Expect(instance.Validate()).To(Succeed())
+			Expect(instance.Init()).To(Succeed())
```

Comment:
What would mean the init?

Reply:
I wasn't very sure about this method's name either. I was also considering calling it `Fill`. I'm open to name suggestions for this method. It basically computes the `Resource` fields that are derived from others.